server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/DataLineJamesMessageHookHandler.java

```diff
@@ -135,6 +134,18 @@ public Response onLine(SMTPSession session, ByteBuffer lineByteBuffer, LineHandl
         return null;
     }
 
+    private byte[] readBytes(ByteBuffer line) {
```

Comment:
there's no existing method to read a bytebuffer to an array?

Reply:
Nope not that I know>

Reply:
me neither